ddl/index.go

```diff
@@ -549,10 +551,26 @@ func (w *addIndexWorker) getIndexRecord(handle int64, recordKey []byte, rawRecor
 	return idxRecord, nil
 }
 
-func (w *addIndexWorker) fetchRowColVals(txn kv.Transaction, taskRange reorgIndexTask) ([]*indexRecord, bool, error) {
+func (w *addIndexWorker) getNextHandle(taskRange reorgIndexTask, handleOutOfRange bool) (nextHandle int64) {
+	nextHandle = taskRange.startHandle
+	if handleOutOfRange {
+		if taskRange.endIncluded && taskRange.endHandle != math.MaxInt64 {
+			nextHandle = taskRange.endHandle + 1
```

Comment:
Could we add a test case to cover it?

Reply:
The test case already added in TestMaxHandleAddIndex.

Reply:
Could we remove this code?

Reply:
nextHandle is used in handleBackfillTask, even if the handleOutRange is true.

Reply:
Got it.